Expired/Views/ProductImage.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  ProductImage.swift
+//  Expired
+//
+//  Created by satgi on 2023-03-22.
+//
+
+import SwiftUI
+
+enum ImageSize {
+    case small
+    case medium
+    case large
+    case ultraLarge
+}
+
+struct ProductImage: View {
+    var image: UIImage? = nil
+    var data: Data? = nil
+    var size: ImageSize = .medium
+
+    var width: CGFloat {
+        var width = 50.0
+        switch size {
+            case .small:
+                width = 50.0
+            case .medium:
+                width = 100.0
+            case .large:
+                width = 150.0
+            case .ultraLarge:
+                width = 250.0
+        }
+
+        return width
+    }
+    
+    var displayImage: UIImage? {
+        if let image = image {
+            return image
+        } else if let data = data {
+            return UIImage(data: data)!
+        }
+
+        return nil
+    }
+
+    var body: some View {
+        if displayImage != nil {
+            Image(uiImage: displayImage!)
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+                .frame(width: width, height: width, alignment: .center)
+                .clipShape(RoundedRectangle(cornerRadius: 4.0))
+                .clipped()
+        } else {
+            Image(systemName: "photo")
+                .font(.system(size: width / 1.18))
+                .frame(width: width, height: width, alignment: .center)
```

Comment:
```suggestion
                .frame(width: width, alignment: .center)
```

We can now remove the `height` parameter since we are using the width variable on the same one.  Having just the width parameter it still sets the width of the image to the width variable and keeps the aspect ratio of the image.

Reply:
In this case, since the width/height ratio is fixed as we are using the SF symbol as a placeholder, it makes sense to remove the `height`.
Actually it makes the edit button closer to the bottom right corner, which is better than it used to be. See attached pictures.
Good catch 👍 

![3](https://user-images.githubusercontent.com/122644881/227738794-13ceaad3-c7c9-4e83-b4ff-f4959a7a0d43.jpeg)

![4](https://user-images.githubusercontent.com/122644881/227738795-c7b74a5c-f750-48da-a77d-dcd8240a5312.jpeg)